test/unclogged/core_test.clj

```diff
@@ -15,6 +15,10 @@
   [n]
   (bit-shift-left n 3))
 
+(defn compose-message
+  [int]
+  (str "message-" int))
```

Comment:
I don't believe that this is a legal message. Messages that are pushed to the sink are maps and I *think* they need the body to be specified under the key `message` (seen in use at https://github.com/RackSec/unclogged/blob/master/src/unclogged/core.clj#L40).

Reply:
Oh, this is a great observation, @derwolfe. I know that based on the docs (and my own interaction with manifold streams) that strings are valid input, _however_ it does seem like unclogged has an expected message structure *for the unclogged sink*: a map with a `message` key. I hadn't considered this. Going to give it a try and circle back when I'm done!

Reply:
@derwolfe the message being invalid turned out to be the culprit. Thank you! I updated the tests to compose the message properly.